simplq/src/components/common/LoginButton/index.jsx

```diff
@@ -23,9 +24,10 @@ const LoginButton = () => {
     setLoadingIndicator(false);
   };
 
-  const onFailureCallback = () => {
+  const onFailureCallback = (resp) => {
+    Sentry.captureMessage(resp);
     Auth.logOut();
```

Comment:
Why add sentry here? Do we really need to be notified of every login failure?

Reply:
I don't know for sure. I get what you mean. Maybe we can collect this and aggregate at sentry level. Something like, define an alert if more than 30% of last 10 logins failed, trigger a notification, as it could be something wrong at our end.

Reply:
That's a good idea. Gotta check if Sentry supports it.